runtime/themes/monokai_soda.toml

```
@@ -0,0 +1,122 @@
+# Monokai Soda port for Helix (https://helix-editor.com)
+# Author : Jimmy Zelinskie <jimmy@zelinskie.com>
+
+# Syntax
+
+## Constants
+"constant" = "white"
+"constant.builtin" = "pink"
+"constant.character.escape" = "blue"
+"constant.numeric" = "purple"
+
+## Diagnostics
+"diagnostic" = { modifiers = ["underlined"] }
+"diagnostic.error" = { underline = { style = "curl", color = "pink" } }
+"diagnostic.warning" = { underline = { style = "curl", color = "orange" } }
+"diagnostic.info" = { underline = { style = "curl", color = "white" } }
+
+## Diffs
+"diff.plus" = "green"
+"diff.delta" = "orange"
+"diff.minus" = "pink"
+
+## Functions
+"function" = "green"
+"function.macro" = "blue"
+"function.builtin" = "pink"
+"constructor" = "blue"
+
+## Git
+"git.delta.moved" = "orange"
```

```suggestion
"diff.delta.moved" = "orange"
```

we don't have anything namespaced under `git` at the moment. I see this key in the nord theme too but I think it's a type for this key. `diff.delta.moved` is used in git commit message highlighting for files that were renamed